site/content/en/docs/Installation/upgrading.md

```diff
@@ -128,7 +128,78 @@ upgrades.
    between the Agones controller being recreated and GameServers being deleted doesn't occur, and GameServers can end up stuck in erroneous states.
 1. Start and complete your control plane upgrade(s).
 1. Start and complete your node upgrades.
-1. Scale your Fleets back up and/or recreate your GameServers. 
+1. Scale your Fleets back up and/or recreate your GameServers.
 1. Run any other tests to ensure the Agones installation is still working as expected.
 1. Close your maintenance window.
 7. Congratulations - you have now upgraded to a new version of Kubernetes! 👍
+
+{{% feature publishVersion="1.41.0" %}}
+## SDK Compatibility Guarantees
+In order to allow compatibility between game server binaries and `sdk-server`, a game server binary
```

Comment:
I was going back through the RFC - and this is probably a better paragraph to start with, then we can get into the details:
https://github.com/googleforgames/agones/issues/3766#client-sdks